tests/integration/states/test_public_ip_address.py

```diff
@@ -145,3 +145,30 @@ def test_absent(salt_call_cli, public_ip_addr, resource_group, connection_auth):
     assert data["changes"]["new"] == expected["changes"]["new"]
     assert data["changes"]["old"]["name"] == expected["changes"]["old"]["name"]
     assert data["result"] == expected["result"]
+
+
+@pytest.mark.run(order=-3)
+def test_absent_second_ip(salt_call_cli, public_ip_addr2, resource_group, connection_auth):
```

Comment:
**comment (non-blocking):** just making the comment here to memorialize that the second IP now needs to be cleaned up since we're re-using the same resource group over and over for testing. before, this was handled by removing the resource group as the final test.